### PR TITLE
Add florence2 aliases and bugfix TransformersModel

### PR DESCRIPTION
**Pull request by probicheaux**
# Description

Add aliases for uploaded base florence2 models and bugfix the base transformers class not using the right processor class

Looks like supervision removed `from_roboflow` so I had to pin supervision low for these tests to pass

Note: right now if someone uses the base model and then tries to use a LoRA, it will have to redownload the base model to a new place for the LoRA :/

## Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] This change requires a documentation update

## How has this change been tested, please provide a testcase or example of how you tested the change?

Locally

## Any specific deployment considerations

n/a

## Docs

-   [ ] Docs updated? What were the changes:

### CODE REVIEW

File: requirements/_requirements.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ prometheus-fastapi-instrumentator<=6.0.0
 redis<6.0.0
 requests>=2.26.0
 rich<=13.5.2
-supervision>=0.20.0,<1.0.0
+supervision>=0.20.0,<=0.21.0
```
Comment (grzegorz-roboflow) [inline requirements/_requirements.txt:13]: this change seems to be not related with the florence2 aliases, do we need this change here?

**Review by grzegorz-roboflow — CHANGES_REQUESTED** [review]

File: requirements/_requirements.txt (hunk 1 above)
Comment (probicheaux) [reply requirements/_requirements.txt:13]: As I said in the description of this pr:

> Looks like supervision removed from_roboflow so I had to pin supervision low for these tests to pass

I have a separate pr fixing the issues in the tests so we can remove the pin

Comment (grzegorz-roboflow) [reply requirements/_requirements.txt:13]: I understand but if we merge this change the #526 is not changing pin in requirements

**Comment by PawelPeczek-Roboflow** [comment]
@probicheaux two things, apart from that it is ok:
* add tests to suite https://github.com/roboflow/inference/blob/main/tests/inference/integration_tests/test_florence.py
* add more tests to https://github.com/roboflow/inference/blob/main/tests/inference/integration_tests/test_paligemma.py to cover other models
* merge with `main` to get requirements

File: requirements/_requirements.txt (hunk 1 above)
Comment (probicheaux) [reply requirements/_requirements.txt:13]: Ah yep it's merged, I'll merge master in here and remove the requirements change

**Review by grzegorz-roboflow — APPROVED** [review]

**Comment by PawelPeczek-Roboflow** [comment]
Tests fail: https://github.com/roboflow/inference/actions/runs/9954485508/job/27500207491
probably to many big models are being pulled

**Review by PawelPeczek-Roboflow — APPROVED** [review]
